index.bs

```diff
@@ -8804,7 +8798,7 @@ the [=named property getter=].
 
 The [{{Global}}] [=extended attribute=] must not
 be used on an [=interface=] that can have more
-than one object implementing it in the same ECMAScript global environment.
+than one object implementing it in the same [=Realm=].
```

Comment:
Above this, [Global] says "it indicates that objects implementing this interface can be used as the global object in an ECMAScript environment". Probably "a Realm" or "an ECMAScript Realm".